README.md

```diff
@@ -61,35 +61,6 @@ If you are using the docker-compose file above, you can use this command to trig
 docker compose exec dmarc-report /usr/bin/dmarcts-report-parser.pl -i -d -r=1
 ```
 
-
-## Optional extended configuration
```

Comment:
Please keep the heading and just add a little link to env.example for the extended configuration options, otherwise I'm fine keeping the rest in this file

Reply:
Done.